Detect new browsers:  Vertex Surf, InBrowser, Every Browser,  World Browser, Holla Web Browser, Tor Browser

sanchezzzhak opened the pull request.

sanchezzzhak committed:
feat(browser): detect World Browser - Fast & Simple
feat(browser): detect Every Browser

ref #7039

sanchezzzhak committed:
feat(browser): detect Insta Browser (com.insta.browser)
feat(browser): detect InBrowser (nu.tommie.inbrowser)
feat(browser): detect Vertex Surf (com.alohamobile.vertexsurf)
feat(app): detect BOX Video Downloader (com.box.video.downloader)
feat(browser): detect MIUI Browser (com.mi.globalbrowser)
feat(browser): detect Puffin Web Browser (com.cloudmosa.puffin)

ref #7039

sanchezzzhak committed:
feat(browser): detect Holla Web Browser (com.hollabrowser.meforce)

ref #7039

sanchezzzhak committed:
fix: phpcs

Simbiat commented on Tests/Parser/Client/fixtures/browser.yml:
```diff
@@ -7639,3 +7639,100 @@
+  user_agent: Mozilla/5.0 (Linux; Android 10; SM-J600G Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/81.0.4044.138 Mobile Safari/537.36
+  client:
+    type: browser
+    name: World Browser - Fast & Simple
```
Would not "World Browser" suffice? "Fast & Simple" feels unnecessary.

sanchezzzhak replied:
Renamed, thank you.

sanchezzzhak committed:
feat(browser): detect Anka Browser Video (com.anka.browser)
feat(browser): detect Dark Browser (com.darkbrowser)
feat(browser): detect Cave Browser (com.cavebrowser)
feat(browser): detect Freedom Browser (freedom.theanarch.org.freedom)
feat(browser): detect MarsLab Web Browser (com.marslab.browserz)
feat(browser): detect Sunflower Browser (com.mini.web.browser)
feat(browser): detect Tor Browser (org.torproject.torbrowser)
feat(browser): detect Zordo Browser (com.zordo.browser)

ref #7039

sanchezzzhak committed:
feat(browser): detect Anka Browser (com.anka.browser)

sanchezzzhak committed:
fix test

sgiehl committed:
Merge branch 'master' into pr_app_001
